src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
do not reset such values in `*` selector - use it for setting `box-sizing`
if you want to reset some default styles do it for a list of tags you really need, i. e. 
```
div,
body {
  margin: 0
}
```